### PR TITLE
Vault integration: use personal auth tokens

### PR DESCRIPTION
**Pull request by czosel**

### CODE REVIEW

File: backend/src/vault/vault-list.js [hunk 1]
```diff
@@ -13,9 +12,10 @@ function stripPrefix(path) {
   return path
 }
 
-async function listVault(path) {
+async function listVault(token, path) {
+  console.log('LIST VAULT', token)
```
Comment (kaldras) [inline backend/src/vault/vault-list.js]: Leftover from testing?

File: backend/src/vault/vault-list.js [hunk 2]
```diff
@@ -13,30 +13,34 @@ function stripPrefix(path) {
...
+      await Promise.all(
+        resp.data.keys.filter(key => key.endsWith('/')).map(async key => {
+          result.children[key] = await listVault(token, path + key)
+        })
+      )
+    }
+
+    return result
+  } catch (e) {
+    throw e
```
Comment (kaldras) [inline backend/src/vault/vault-list.js]: What is the use of this `catch`?

Comment (czosel) [inline backend/src/vault/vault-list.js]: There's no use for it, i misinterpreted a node error - thanks!

File: backend/src/vault/vault-list.js (hunk 1 above)
Comment (czosel) [inline backend/src/vault/vault-list.js]: Yep, should be gone already.

**Pull request merged**